Controller/SettingsController.php

```diff
@@ -24,7 +24,10 @@ public function modifyAction(Request $request) {
 			'settings' => $allStoredSettings,
 		);
 
-		$form = $this->createForm(new ModifySettingsForm(), $formData);
+		$form = Kernel::VERSION_ID < 20800
+			? $this->createForm(new LegacyModifySettingsForm(), $formData)
```

Comment:
Done in #23. I've added an alias for this form as well. There was only one for the other type (being 2 in total).